packages/cxl-ui/scss/cxl-featured-image.scss

```diff
@@ -0,0 +1,25 @@
+:host {
+  position: relative;
+  display: block;
+  min-width: 320px;
+  height: 25vh;
+
+  #container {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 75%;
+    background-color: var(--cxl-featured-image-bg-color, var(--cxl-brand-color-black, rgb(26, 26, 26)));
```

Comment:
We do not have `--cxl-featured-image-bg-color` defined anywhere so component background color is not compatible with what we see on the desktop.

What about refactoring this code: https://github.com/conversionxl/aybolit/blob/22f622517eb40db969d95d1300c587743fd1de65/packages/cxl-ui/scss/cxl-section.scss#L119-L147

We could set variable value instead of explicit background color value. Doable @anoblet?